project_resstock_national/scripts/analysis/finalize.args

```diff
@@ -1 +1,4 @@
-[]
\ No newline at end of file
+[
+  "\"BASE\", \"Triple-Pane Windows\"",
+  "\"BASE\", \"R-13 Wall Insulation\""
+]
```

Comment:
I thought we decided to leave these blank so that it would default to calculating all upgrades against BASE in case users change the upgrades in the project. I could go either way...

Reply:
You're right. I forgot to remove them after testing.